client/app/pages/queries/view.js

```diff
@@ -440,6 +440,19 @@ function QueryViewCtrl(
     });
   };
 
+  $scope.openAddToDashboardForm = (visId) => {
+    const visualization = getVisualization(visId);
+    $uibModal.open({
+      component: 'addToDashboardDialog',
+      size: 'sm',
+      resolve: {
+        query: $scope.query,
+        vis: visualization,
+        saveAddToDashboard: () => $scope.saveAddToDashboard,
```

Comment:
I couldn't find where this is used to where `$scope.saveAddToDashboard` is defined. Maybe a leftover?